lib/shaman.rb

```diff
@@ -16,5 +15,5 @@ module Shaman
     else
       'https://infinum.tryoutapps.com'
     end
-  LABS_URL = ENV['SERVER_URL'] || DEFAULT_URL
+  LABS_URL = ENV.fetch('SERVER_URL', DEFAULT_URL)
```

Comment:
Can we fix the references from `LABS` to `TRYOUTAPPS` if there aren't too many?

Reply:
Added a todo under [refactoring task](https://app.productive.io/1-infinum/projects/134/tasks/task/7565017?filter=MjQxOTMy).